doc/jdbc-connections.md

```diff
@@ -0,0 +1,41 @@
+# JDBC Connections Configuration
+This is a file containing the JDBC Connections that are available in the gateway.
```

Comment:
Not sure if we should put export documentation here, but we should mention that the jdbc will only be exported when it's referenced by a policy

Reply:
I think we need to add documentation for the export plugin there, not here.